lib/__tests__/standalone-globs.test.js

```diff
@@ -112,6 +112,35 @@ describe('standalone globbing', () => {
 		);
 	});
 
+	describe('windows path style with backslash', () => {
+		// The tests fail on non-windows environments.
+		if (process.platform !== 'win32') {
+			return;
+		}
+
+		it('should handle normal backslash in glob pattern', async () => {
```

Comment:
The description seems inappropriate because this test no longer uses a glob pattern.
For example, how about fixing it as below?

```suggestion
		it('should handle a backslash-separated file path', async () => {
```